sm/plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -84,10 +87,18 @@ fn run(operation: PluginOp) -> Result<std::process::ExitStatus, InternalError> {
             }
         }
 
-        PluginOp::Remove {
-            module,
-            version: _unused,
-        } => run_cmd("apt-get", &format!("remove --quiet --yes {}", module))?,
+        PluginOp::Remove { module, version } => {
+            if let Some(version) = version {
+                // check the version mentioned present or not
+                if check_if_the_module_with_version_installed(&module, &version)? {
```

Comment:
This check is useless, since the `apt-get remove` command accepts a version as the  `apt-get install` command.

e.g.

```
sudo apt-get remove sqlite3=3.27.2-3+deb10u1
```

Reply:
hmm, fixed it.